types/stylelint/index.d.ts

```diff
@@ -1,37 +1,56 @@
-// Type definitions for stylelint 9.4
+// Type definitions for stylelint. 10.0
```

Comment:
There is no such stylelint version.

Reply:
@sandersn This is correct change? The change is breaking change but library hasn't moved to new version

Reply:
I vote for an existing minor version, such as 9.5.